dojo/tools/neuvector/parser.py

```diff
@@ -0,0 +1,117 @@
+import json
+import logging
+
+from dojo.models import Finding
+
+logger = logging.getLogger(__name__)
+
+NEUVECTOR_SCAN_NAME = 'NeuVector (REST)'
+NEUVECTOR_IMAGE_SCAN_ENGAGEMENT_NAME = 'NV image scan'
+NEUVECTOR_CONTAINER_SCAN_ENGAGEMENT_NAME = 'NV container scan'
+
+
+class NeuVectorJsonParser(object):
+    def parse(self, json_output, test):
+        tree = self.parse_json(json_output)
+        items = []
+        if tree:
+            items = [data for data in self.get_items(tree, test)]
+        return items
+
+    def parse_json(self, json_output):
+        try:
+            data = json_output.read()
+            try:
+                tree = json.loads(str(data, 'utf-8'))
+            except:
+                tree = json.loads(data)
+        except:
+            raise Exception("Invalid format")
+
+        return tree
+
+    def get_items(self, tree, test):
+        items = {}
+        if 'report' in tree:
+            vulnerabilityTree = tree['report'].get('vulnerabilities', [])
+            for node in vulnerabilityTree:
+                item = get_item(node, test)
+                package_name = node['package_name']
+                if len(package_name) > 64:
+                    package_name = package_name[-64:]
+                unique_key = node['name'] + str(package_name + str(
+                    node['package_version']) + str(node['severity']))
+                items[unique_key] = item
+        return list(items.values())
+
+
+def get_item(vulnerability, test):
+    severity = convert_severity(vulnerability['severity']) if 'severity' in vulnerability else "Info"
+    vector = vulnerability['vectors_v3'] if 'vectors_v3' in vulnerability else "CVSSv3 vector not provided. "
+    fixed_version = vulnerability['fixed_version'] if 'fixed_version' in vulnerability else "There seems to be no fix yet. Please check description field."
```

Comment:
```suggestion
    fixed_version = vulnerability.get('fixed_version', 'There seems to be no fix yet. Please check description field.')
```
Something like this across the file would be much cleaner

Reply:
Thank you, I've pushed the updated version.